src/pages/Explore/Explore.tsx

```diff
@@ -25,18 +46,23 @@ const activeCard = {
   opacity: 0.5,
 }
 
+const linkIcon = <RiArrowRightUpLine />
+
 const ExploreCard: React.FC<ExploreCardProps> = props => {
   const { title, body, icon, ...rest } = props
   return (
     <Card _active={activeCard} {...rest}>
       <CardBody display='flex' flexDir='column' alignItems='flex-start'>
-        <Center fontSize='4xl' width='auto' mb={6}>
+        <Center fontSize='4xl' width='auto' mb={2} opacity={0.3}>
```

Comment:
suggestion: float as string

Reply:
Done in [6f43640](https://github.com/shapeshift/web/pull/10191/commits/6f43640b9aa2cf8b0364dc9b591858409921a466)